src/scss/styles.scss

```diff
@@ -0,0 +1,119 @@
+@mixin margins($values...) {    
+    @each $var in $values {
+        margin: #{$var};
+    }
+}
+
+@mixin font($family, $size, $weight, $style, $color, $textAlign) {
```

Comment:
this is confusing
better to have smaller multiple mixins or make a smaller mixin and keep the other styles specific to the areas they are needed